src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/DefaultTranscriptsList.scss

```diff
@@ -0,0 +1,23 @@
+@import 'src/styles/common';
+@import 'constants';
+
+.defaultTranscriptsList {
+  position: relative;
+}
+
+.stripedBackground {
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: $left_column;
+  right: 0;
+
```

Comment:
Is the position styling needed here? I don't see any change in the UI when all of them are disabled.

Reply:
`right` is not needed, that's true. Is this what you mean?

Reply:
I meant all of them. But as we discussed offline, it is only the `right` that is not needed.

Reply:
✔️